app/models/foreman_google/gce.rb

```diff
@@ -85,10 +85,14 @@ def destroy_vm(uuid)
       true
     end
 
-    def available_images(filter: nil)
+    def available_images(filter: filter_for_images)
       client.images(filter: filter)
     end
 
+    def filter_for_images
```

Comment:
Is this expected to be overridden from somewhere?

Reply:
Yop, it's overridden from the `foreman_theme_satellite` plugin